airbyte-integrations/connectors/source-postgres/metadata.yaml

```diff
@@ -6,7 +6,7 @@ data:
   connectorSubtype: database
   connectorType: source
   definitionId: decd338e-5647-4c0b-adf4-da0e75f5a750
-  dockerImageTag: 3.1.13
```

Comment:
💅 This should be a minor instead of a patch

Reply:
+1

Reply:
+1

Reply:
Done